shared/src/domain-logic/determineForeignBusinessType.ts

```diff
@@ -0,0 +1,19 @@
+import { ForeignBusinessTypeId } from "../profileData";
+
+export type ForeignBusiness = "REMOTE_WORKER" | "REMOTE_SELLER" | "NEXUS" | "NONE" | undefined;
+
+export const determineForeignBusinessType = (id: ForeignBusinessTypeId[]): ForeignBusiness => {
```

Comment:
[sand/pebble] We could refactor this function. It would be cool as a switch statement.

Reply:
I agree with that. I think this is a great candidate for a `switch`.

Reply:
[sand] The param we're using here should be plural. When I see `id` it implies to me that we're looking at a single id, not an array. Despite the typing being present, I got confused when looking at the rest of the function.